Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.0.1'
```

Comment:
Hmmm seems like theres been a bunch of debate, a few patches, some more debate, and the community has settled on no versions in Gemfile. TIL